src/status_im/common/standard_authentication/password_input/view.cljs

```diff
@@ -21,31 +21,62 @@
     (i18n/label :t/oops-wrong-password)
     error))
 
-(defn- on-change-password
-  [entered-password]
-  (debounce/debounce-and-dispatch [:profile/on-password-input-changed
-                                   {:password (security/mask-data entered-password)
-                                    :error    ""}]
-                                  100))
+(defn- error-info
+  [error-message processing shell?]
+  (let [theme    (quo.theme/use-theme-value)
+        on-press (rn/use-callback
+                  (fn []
+                    (rn/dismiss-keyboard!)
+                    (rf/dispatch [:show-bottom-sheet
+                                  {:content #(forgot-password-doc/view {:shell? shell?})
+                                   :theme   theme
+                                   :shell?  shell?}]))
+                  [theme])]
+    [rn/view {:style style/error-message}
+     [quo/info-message
+      {:type :error
+       :size :default
+       :icon :i/info}
+      error-message]
+     [rn/pressable
+      {:hit-slop {:top 6 :bottom 20 :left 0 :right 0}
+       :disabled processing
+       :on-press on-press}
+      [rn/text
+       {:style                 {:text-decoration-line :underline
+                                :color                (colors/resolve-color :danger theme)}
+        :size                  :paragraph-2
+        :suppress-highlighting true}
+       (i18n/label :t/forgot-password)]]]))
 
-(defn- view-internal
-  [{:keys [default-password theme shell? on-press-biometrics blur?]}]
+(defn view
+  [{:keys [shell? on-press-biometrics blur?]}]
   (let [{:keys [error processing]} (rf/sub [:profile/login])
-        error-message              (get-error-message error)
-        error?                     (boolean (seq error-message))]
+        error-message              (rn/use-memo #(get-error-message error) [error])
+        error?                     (boolean (seq error-message))
+        default-value              (rn/use-ref-atom "") ;;bug on Android
+                                                        ;;https://github.com/status-im/status-mobile/issues/19004
+        on-change-password         (rn/use-callback
+                                    (fn [entered-password]
+                                      (reset! default-value entered-password)
+                                      (debounce/debounce-and-dispatch [:profile/on-password-input-changed
+                                                                       {:password (security/mask-data
+                                                                                   entered-password)
+                                                                        :error    ""}]
+                                                                      100)))]
     [:<>
      [rn/view {:style {:flex-direction :row}}
       [quo/input
        {:container-style {:flex 1}
         :type            :password
+        :default-value   @default-value
```

Comment:
this is the only related change, other just slight adjusting according new UI guidelines